test/pbt/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+set(COMPILER_ARGS
+    --std=c++${CMAKE_CXX_STANDARD}
+    -Wall
+    -Wcast-align
+    -Wconversion
+    -Wdouble-promotion
+    -Werror
+    -Wextra
+    -Wextra-semi
+    -Wfatal-errors
+    -Wformat=2
+    -Wold-style-cast
+    -Woverloaded-virtual
+    -Wshadow
+    -Wunused
+    -Wno-missing-braces)
```

Comment:
**suggestion**: We can DRY this with something like:
```cmake
get_target_property(common_warnings warnings INTERFACE_COMPILE_OPTIONS)
list(TRANSFORM common_warnings GENEX_STRIP OUTPUT_VARIABLE common_warnings)
list(REMOVE_ITEM common_warnings "")
```
We can do similar for the `INTERFACE_COMPILE_OPTIONS` of `stdx`.
Then add the standard option.